### PR TITLE
Update helpView.ts for changes to organization name

### PR DESCRIPTION
**Pull request by worksofliam**
### Changes

Description of change here.

### Checklist

* [x] have tested my change
* [x] updated relevant documentation
* [ ] Remove any/all `console.log`s I added
* [ ] eslint is not complaining
* [ ] have added myself to the contributors' list in [CONTRIBUTING.md](https://github.com/halcyon-tech/vscode-ibmi/blob/master/CONTRIBUTING.md)
* [ ] **for feature PRs**: PR only includes one feature enhancement.

### CODE REVIEW

**Review by sebjulliand — CHANGES_REQUESTED** [review]
The change looks good.
But while you're at it, why not replace **all** references to `halcyon-tech` by `codefori`?

By the look of it, a simple replace all should do the trick 😄 (in README, CONTRIBUTING, package, pr.yaml, etc)

**Comment by worksofliam** [comment]
@sebjulliand Didn't think about that! Please check again :)

File: package.json [hunk 1]
```diff
@@ -14,11 +14,11 @@
 	],
 	"author": {
 		"name": "Halcyon Tech Ltd",
```
[the remark this answers is not in the document]
Comment (worksofliam) [reply package.json:16]: Whoops!! Didn't look for strings like that!

**Review by sebjulliand — CHANGES_REQUESTED** [review]
Looks good. There are two places where Halcyon remains and may be changed:
- In `package.json` (see suggestion)
- In `LICENSE`, it says `Copyright (c) 2022 Halcyon Tech Ltd`; should it be `Copyright (c) 2023 Code4i` now? (year is wrong too 😅)

**Comment by sebjulliand** [comment]
> @sebjulliand Done!

Nice! But don't forget to push 😁

**Comment by chrjorgensen** [comment]
@worksofliam Is there a redirect from the old name to the new one? Or are all links in issues etc. now broken? Noticed that Scott referenced the old name in his presentation at the IBM Champions IBM i Updates briefing...

**Comment by worksofliam** [comment]
@chrjorgensen The following link still works, but I am guessing it won't by the time next week comes around.

https://github.com/halcyon-tech/vscode-ibmi

**Comment by worksofliam** [comment]
@sebjulliand I pushed. I had a weird situation where my remote broke right in the middle of this PR (still using the old `halcyon-tech` namespace). Fixed it now and pushed up.

**Review by sebjulliand — APPROVED** [review]
Approved and merged! Good job @worksofliam !

**Pull request merged**